common-types/package.json

```diff
@@ -8,4 +8,4 @@
     "build": "exit 0"
   },
   "dependencies": {}
-}
+}
```

Comment:
Can you get rid of the extra line?

Reply:
Its weird that that's showing, but on my end there is no extra line, just 11

Reply:
I think if you just do `git checkout main common-types/package.json` it should revert the change